app/lib/shared/markdown.dart

```diff
@@ -53,6 +53,12 @@ String markdownToHtml(
       nodes = _groupChangelogNodes(nodes).toList();
     }
     return _renderSafeHtml(nodes, disableHashIds: disableHashIds);
+  } catch (e, st) {
+    _logger.shout('Error rendering markdown.', e, st);
+    // disabling the opening and closing tags makes sure we don't have any
+    // HTML elements that would break out of the wrapper <pre>
+    final safeText = text.replaceAll('<', ' ').replaceAll('>', ' ');
```

Comment:
Could we replace with `&lt;` and also replace `&` with &amp; 

See https://www.geeksforgeeks.org/which-characters-should-be-escaped-inside-a-pre-tag/

Reply:
Done.